core/jreleaser-engine/src/main/java/org/jreleaser/engine/context/ModelAutoConfigurer.java

```diff
@@ -482,6 +479,10 @@ private JReleaserModel autoConfiguredModel(Path basedir) {
             Repository repository = GitSdk.of(basedir, grs).getRemote();
             BaseReleaser<?, ?> service = null;
             switch (repository.getKind()) {
+                case BITBUCKET_CLOUD:
+                    service = new BitbucketCloudReleaser();
+                    model.getRelease().setBitbucketCloud((BitbucketCloudReleaser) service);
+                    break;
```

Comment:
this was not intuitive, we should figure out how to make it "type safe" and part of the api